wasm/jit/jit_amd64_test.go

```diff
@@ -735,6 +747,49 @@ func TestAmd64Builder_handlePick(t *testing.T) {
 	})
 }
 
+func TestAmd64Builder_handleConstI32(t *testing.T) {
+	for _, v := range []uint32{1, 1 << 5, 1 << 31} {
+		t.Run(fmt.Sprintf("%d", v), func(t *testing.T) {
+			o := &wazeroir.OperationConstI32{Value: v}
```

Comment:
maybe move this down to "emit the const.." because I was wondering what relationship this has to the comment "dummy value not used"

Reply:
https://github.com/tetratelabs/wazero/pull/82/commits/d0f954e07906a5a8e03a7fae7dbc71c43ff8c8dc